e2e/e2etests/e2etests.go

```diff
@@ -955,13 +955,17 @@ var AllE2ETests = []runner.E2ETest{
 	runner.NewE2ETest(
 		TestV2ERC20WithdrawAndCallName,
 		"withdraw ERC20 from ZEVM and authenticated call a contract using V2 contract",
-		[]runner.ArgDefinition{},
+		[]runner.ArgDefinition{
+			{Description: "amount", DefaultValue: "1000"},
```

Comment:
Nit: it was 10k (`big.NewInt(10000)`)

Reply:
Just to make the same as other test, the amount itself is not used in the function in fact